.github/CODEOWNERS

```diff
@@ -12,9 +12,34 @@
 # Make sure that team names are correct or everything stops working.
 # Organisation name is @island-is, not @island.is
 
-
 /libs/api/domains/identity/                                                                   @island-is/core
 
+/apps/reference-next-app/                                                                     @island-is/aranja-core
+/libs/api/domains/translations/                                                               @island-is/aranja-core
+/libs/contentful-extensions/                                                                  @island-is/aranja-core @island-is/kosmos-kaos
+/libs/contentful-extensions/translation/                                                      @island-is/aranja-core
+/libs/localization/                                                                           @island-is/aranja-core
+/libs/utils/api/                                                                              @island-is/aranja-core
+/libs/utils/shared/                                                                           @island-is/aranja-core
+/libs/service-portal/settings/access-control                                                  @island-is/aranja-core @island-is/vice-versa
```

Comment:
I guess we lose significance here then instead. Should maybe have "top level" project ownership first (service-portal, application-system, web and api), then all the rest grouped by teams?

Reply:
Sounds good to me. I wasn't sure of the best way to go with this. Will update the PR 👍

Reply:
Take 2 is here https://github.com/island-is/island.is/pull/4367